debug/gdbserver.py

```diff
@@ -1039,8 +1039,21 @@ def test(self):
             local = self.gdb.p("local")
             if interrupt_count > 1000 and \
                     local > 1000:
+                # Disable timer interrupt
+                for hart in self.target.harts:
+                    self.gdb.select_hart(hart)
+                    self.gdb.p("$mie")
+                    self.gdb.p("$mie=$mie & ~0x80")
+                    self.gdb.p("$mie")
```

Comment:
Can you make this a helper function in GdbTest, and call that from the various places that need it?

You can probably remove the extra prints of $mie, too.

Reply:
I have made the changes as you suggested and conducted debug testing.